### DISCUSSION TITLE
LocalGPT as an extension for oobabooga

### DISCUSSION THREAD

**Discussion by LitenBuzzTh** [PromtEngineer/localGPT#164] [Ideas]
Hi, Today I was experimenting with "superbooga" which is an extension for oobabooga, which is a littlebit similar to localgpt.
I wondered If there it could be a good Idea to make localGPT able to be installed as an extension for oobabooga.

**Comment by PromtEngineer** [comment]
@LitenBuzzTh I like the idea but will need support. I don't have the expertise to convert this into an extension. If someone can help, I would love to do that.
